nodesetup.sh

```diff
@@ -1,7 +1 @@
-npm install -g eslint
-npm install -g grunt-cli
-npm install -g csslint
-npm install -g jshint
-npm install -g nodemon
-npm install -g forever
-npm install -g npm
+npm install -g eslint grunt-cli csslint jshint nodemon forever npm
```

Comment:
Hello David,
I've intentionally written the node modules on separate lines cause I find it easier to read and scan through and work with.

And since performance does not matter here I will stick with that solution.